docs/api-reference/mode-handlers/overview.md

```diff
@@ -72,6 +72,12 @@ User can draw a new circular `Polygon` feature by clicking the center then along
 
 User can draw a new circular `Polygon` feature by clicking the two corners of bounding box.
 
+### ModeConfig
+
+A `modeConfig` object can be provided:
+
+* If options: `{steps: x}` means the circle will be drawn using `x` number of points.
```

Comment:
It's not clear if the provided object is `modeConfig: { options: { steps: 24 } }` or `modeConfig: { steps: 24 }`. Perhaps look at how `booleanOperation` is documented.

Reply:
And if `modeConfig: { options: { steps: 24 } }` is currently implemented, I think "config" and "options" is redundant. `modeConfig: { steps: 24 }` is simpler.

Reply:
Options was added incase if there was a need to pass multiple configuration such as steps, unit etc rather than having a flattened object. 

Changed it for simplicity now as `modeConfig: { steps: 24 }`